conf/base_config.yaml

```diff
@@ -163,4 +163,14 @@ predict:
     encoder: ${encoder}
     weight_save_path: ${paths.pretrained_models}/sdss.pt
     device: "cuda:0"
-    crop: [0, -1]
+    crop:
+        if_crop: true
+        left_upper_corner: [160, 160]
+        width: 640
+        height: 640
+    plot:
+        if_plot: true
```

Comment:
Would it make sense to rename `if_plot` to `show_plot`? The variable name `if_plot` doesn't seem self-explanatory, but `show_plot` makes sense to me right away (if that's indeed what this boolean variable controls)

Reply:
Same for `if_crop`, I think I'd called it `do_crop` or just `crop`.